compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -42,8 +42,8 @@ impl<'tcx> Context for Tables<'tcx> {
         self.tcx.def_path_str(self[def_id])
     }
 
-    fn span_of_an_item(&mut self, def_id: stable_mir::DefId) -> stable_mir::ty::Span {
-        self.span(self.tcx.def_span(self[def_id]))
+    fn span_of_an_item(&mut self, def_id: stable_mir::DefId) -> String {
+        opaque(&self.tcx.def_span(self[def_id])).to_string()
```

Comment:
Sorry. The confusion was my inability to use github correctly.

Lemme try again:

With this change, there is no way to obtain a span anymore. Before this PR this function returned `Opaque`, now it returns `String`, which is the same thing.

If I understand it correctly, this commit is because you were unable to meaningfully debug print `Span`. If you wanna fix that, I think, instead of this commit, you should do something similar to `Span`'s `Debug` impl as was done to `DefId`'s `Debug` impl